inc/computer_softwareversion.class.php

```diff
@@ -827,7 +882,60 @@ class='submit'>";
       }
       echo "<div class='spaced'>";
       // Affected licenses NOT installed
-      //needs DB::request() to support aliases to get migrated
+      $lic_where = [];
```

Comment:
Do not see where it is used.

Reply:
Should be in the commented iterator (will fix)